src/fsharp/TypeChecker.fs

```diff
@@ -7684,14 +7684,22 @@ and TcComputationExpression cenv env overallTy mWhole (interpExpr: Expr) builder
     let customOperationMethodsIndexedByKeyword = 
         customOperationMethods
         |> Seq.groupBy (fun (nm, _, _, _, _, _, _, _, _) -> nm)
-        |> Seq.map (fun (nm, g) -> (nm, Seq.toList g))
+        |> Seq.map (fun (nm, g) ->
```

Comment:
Thanks for following the conventions of the codebase as they are today.

However, I think it would improve everything if `g` were renamed to `group`. The reason is because `g` is almost universally used throughout the compiler to refer to an instance of `TcGlobals`. You get trained very quickly to see a `g` this way, much like you get trained to see `m` as a `range`. The existing code violated this pattern and for existing contributors it could throw them off a bit.

Reply:
Okay, g to group and nm to nrange? Or is no okay as is?